benchmark/include/util/storage_benchmark_util.h

```diff
@@ -22,30 +22,22 @@ struct TupleAccessStrategyBenchmarkUtil {
     for (uint32_t i = 0; i < num_bytes; i++) out[i] = static_cast<byte>(dist(*generator));
   }
 
-  // Write the given fake tuple into a block using the given access strategy,
+  // Write the given tuple (projected row) into a block using the given access strategy,
   // at the specified offset
-  static void InsertTuple(const FakeRawTuple &tuple, const storage::TupleAccessStrategy *tested,
+  static void InsertTuple(const storage::ProjectedRow *tuple, const storage::TupleAccessStrategy *tested,
```

Comment:
Are we modifying tuple or tested? Back in #53, we decided to follow [google guidelines](https://google.github.io/styleguide/cppguide.html#Reference_Arguments) on reference arguments.